test/helpers/FormMatchers.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package helpers
+
+import org.scalatest.matchers.{BeMatcher, MatchResult}
+import play.api.data.{Form, FormError}
+
+trait FormMatchers {
+  val errorless: BeMatcher[Form[_]] = new BeMatcher[Form[_]] {
```

Comment:
Nit: It took me a while to realise what it does. I would name sth like `withoutErrors`. But might be just a matter of slow thinking.